aws/aws-cloudwatch-alarm-expression/README.md

```diff
@@ -0,0 +1,45 @@
+# aws-cloudwatch-alarm-expression
```

Comment:
we should remove the `aws-` prefix as we start to scope modules under clouds.

Reply:
Done